index.html

```diff
@@ -8,29 +8,180 @@
+                    <li class="canopy"><h3>CANOPY</h3></li>
+                    <li class="c-text"><h4> Back End Dev</h4></li>
+                    <li class="c-text"><h4> 2015</h5></li>
+                </ul>
+                <div class="card-text">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="languages">
+                    <a href="" class="btn">HTML</a>
+                    <a href="" class="btn">CSS</a>
+                    <a href="" class="btn">JavaScript</a>
+                </div>
+                <div class="project-link">
+                    <a href="" class="btn2">See Project</a>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="about-section">
+        <section class="cont-about">
+            <div class="about-myself">
+                <h1 class="about-title">About Myself</h1>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            </div>
+            <div class="connect">
+                <p>LET'S CONNECT</p>
+            </div>
+            <ul class="media-icon2">
+                <li><a href="" class="icon4"><img src="src/img/icons/Vector (2).png" alt="icon4"></a></li>
+                <li><a href="" class="icon2"><img src="src/img/icons/Linkedin icon.png" alt="Linkedin"></a></li>
+                <li><a href="" class="icon5"><img src="src/img/icons/Vector (3).png" alt="icon5"></a></li> 
+                <li><a href="" class="icon1"><img src="src/img/icons/Vector.png" alt="Twitter"></a></li>
+                <li><a href="" class="icon3"><img src="src/img/icons/Vector (1).png" alt="icon3"></a></li>  
+            </ul>
+            <div class="resume-link">
+                <a href="" class="btn3">Get my resume</a>
+            </div>
+        </section>
+        <div class="bottom-menus">
+            <div class="languages-menu">
+                <div class="languages-des">
+                    <h3>Languages</h3>
+                    <img src="src/img/icons/chevron-down-regular-24.png" alt="des 1">
+                </div>
+                <div class="languages-box">
+                    <div class="javascript-box">
+                        <img src="src/img/icons/Ellipse 1.png" alt="JS-Icon">
+                        <p>JavaScript</p>
+                    </div>
+                    <div class="html-box">
+                        <img src="src/img/icons/Ellipse 1 (1).png" alt="HTML-Icon">
+                        <p>HTML</p>
+                    </div>
+                    <div class="css-box">
+                        <img src="src/img/icons/Ellipse 1 (2).png" alt="CSS-Icon">
+                        <p>CSS</p>
+                    </div>
+                </div>
```

Comment:
- [ ] The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`. Kindly do the change. :+1: